### ISSUE TITLE
WCAG 1.1.1 🟠 (Partially Supports): Non Text Content

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#5049]
## WCAG 1.3.1: Info and Relationships

**Level:** A
**Principle:** Perceivable
**Result:** 🟠 Partially Supports

### Description
This issue tracks the audit and implementation of WCAG 2.2 Success Criterion 1.3.1: Info and Relationships.

### Resources
- [WCAG 2.2 Success Criterion 1.1.1](https://www.w3.org/WAI/WCAG22/Understanding/non-text-content)
> All [non-text content](https://www.w3.org/WAI/WCAG22/Understanding/non-text-content#dfn-non-text-content) that is presented to the user has a [text alternative](https://www.w3.org/WAI/WCAG22/Understanding/non-text-content#dfn-text-alternative) that serves the equivalent purpose, except for ...

### Audit Checklist
- [x] Review previous Audit Result
- [x] Review work identified in previous audit and list open issues.
- [x] Check for any other relevant open issues.
- [x] Retest open issues, are these issues still present?
- [x] Test for any new issues that may have been introduced.
- [x] Determine compliance status, add this to issue name.
- [x] Add any new issues to the Accessibility project for triage.
- [x] Replace previous audit in milestone with this update.

### [Previous audit](https://github.com/openlibhums/janeway/issues/4126)

> ❌ Aside from user-content, the overwhelming majority of front end content is text, with non-text content as decorative. The audit has found some incorrect markup. While this audit excludes User-Content, we have recognised that the majority of non-text content is user-content and so are updating our alt-text support accordingly.

 ### Progress since last audit

#### Incorrect markup

- #3924 
- #4212 
- #4215

#### Captcha
- #4058

#### Alternative Text
- #3943 
- #3938 

#### In Progress 
work on these items almost complete as part of a new alt-text management system. This has been through design, development and intial checks, but will not be merged into the master branch until after the a11y-audit-1.9 tag.
- #2825
- #4213 
- #4217 

 ### Audit Notes
Automated testing of both a11y-audit-1.9 tag and the alt-text branch looking for missing alt-text show some missing text on the Press homepage in the Clean theme, and one item missing on each of three other press pages.  The other themes and Clean Journals pages are all clear. 

 ### Work arising
No further work can be accurately identified for text alternatives until the new alt-text system is in place and we can find out what remains to be done. So while further work is anticipated, it cannot currently be specified.
